### PR TITLE
Implementing NavigateTo function in BrowserChrome + Reuse handle aad AccountPicker for sign out as well

### PR DESCRIPTION
**Pull request by kkrishnareddy**
Implementing NavigateTo function in Browser Chrome. if we are on home screen of tabs in chrome then only it works.

Also refactor handleAccountPicker in AadLoginComponentHandler for sign out as well

### CODE REVIEW

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java [hunk 1]
```diff
@@ -34,6 +36,6 @@
      *
      * @param url the url to open
      */
-    void navigateTo(String url);
+    void navigateTo(String url) throws UiObjectNotFoundException;
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java:39]: I don't think an exception is thrown here....why do we need the throws clause?

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java [hunk 2]
```diff
@@ -46,7 +53,11 @@ public void handleFirstRun() {
     }
 
     @Override
-    public void navigateTo(@NonNull final String url) {
+    public void navigateTo(@NonNull final String url) throws UiObjectNotFoundException {
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java:56]: I don't think an exception is thrown here....why do we need the throws clause?

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java [hunk 3]
```diff
@@ -46,7 +53,11 @@ public void handleFirstRun() {
...
         //TODO: implement browsing for Chrome
```
Comment (shahzaibj) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java:57]: We can remove the TODO statement now

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java (hunk 1 above)
Comment (kkrishnareddy) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/IBrowser.java]: yes, initially I used the .click() method that's why it is causing an exception.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java (hunk 2 above)
Comment (kkrishnareddy) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java]: yes, initially I used the .click() method that's why it is causing an exception.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java (hunk 3 above)
Comment (kkrishnareddy) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserChrome.java]: removed the TODO statement.

**Review by shahzaibj — APPROVED** [review]

**Pull request merged**